modules/admin_manual/pages/maintenance/upgrading/upgrade_php.adoc

```diff
@@ -1,127 +1,45 @@
-= Upgrade PHP on RedHat 7 and Centos 7
+= Upgrade PHP on RedHat 7 and CentOS 7
 :toc: right
+:toclevels: 1
+:keywords: upgrade, red hat, centos
+:description: Upgrade PHP to versions 7.0 - 7.4 on Red Hat and CentOS so that you can make the most out of your ownCloud installations.
```

Comment:
```suggestion
:description: Upgrade PHP to versions 7.0 - 7.3 on Red Hat and CentOS so that you can make the most out of your ownCloud installations.
```
PHP 7.4 support by ownCloud is not "a thing" yet.